styles.css

```diff
@@ -40,59 +35,290 @@ header {
   color: #fff;
-  flex: none;
-  order: 2;
-  flex-grow: 0;
 }
 
 .icon:hover {
   background-color: rgb(132, 132, 148);
 }
 
+.work ul {
+  padding: 1vh 10vh;
+}
+
+.skills li {
+  padding: 2vh 2vh;
+  display: block;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 3.3vh;
+  line-height: 6.5vh;
+  font-weight: 400;
+}
+
+.work ul li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 2.5vh;
+  line-height: 3.04vh;
+  color: #091e42;
+}
+
+.card ul {
+  padding: 0%;
+  margin: 0%;
+  margin-bottom: 3vh;
+}
+
+.card ul li {
+  background: rgba(255, 255, 255, 0.24);
+  margin-left: 0;
+  margin-right: 2vh;
+}
+
 #social-icons ul {
   padding: 0;
 }
+
+.work {
+  background: #fff;
+}
+
+.work h1 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 6.6vh;
+  padding-top: 12vh;
+  color: #1c1a19;
+  text-align: center;
+}
+
+.work h2 {
+  margin-left: 4vh;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 6vh;
+  color: #091e42;
+}
+
+hr {
+  width: 80%;
+  margin-bottom: 5vh;
+}
+
+.work img {
+  padding: 5vh 5vh 0.55vh 5vh;
+  width: 80%;
+}
+
+.work p {
+  text-align: left;
+  padding: 0 5vh;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 2.43vh;
+  line-height: 3.6vh;
+  color: #091e42;
+  width: 49.7vh;
+  height: 24.5;
+}
+
+.language {
+  padding: 1.8vh;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', sans-serif;
+  box-sizing: border-box;
+  align-items: center;
+  font-weight: 500;
+  font-size: 2.2vh;
+  line-height: 3.04vh;
+}
+
+button {
+  color: #fff;
+  background: #ff6b00;
+  border: none;
+  padding: 1vh;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: 0.1vh;
+  text-align: center;
+}
+
+#see-project-small {
+  width: 18.5vh;
+  height: 7.3vh;
+  margin: 0 5vh;
+  font-size: 2.58vh;
+}
+
+button:hover {
+  background-color: #e78139;
+}
+
+.grid-card {
+  display: grid;
+  grid-template-columns: auto;
+}
+
+.wrapper {
+  background-image: url("Images/work1.png");
+  background-position: 100% 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-left: 5vh;
+  margin-right: 12vh;
+  padding-top: 15vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.card {
+  color: #fff;
+  padding-left: 2vh;
+}
+
+.card h1 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 4.8vh;
+  line-height: 6.6vh;
+  width: 44.9vh;
+  height: 12.3vh;
+  padding-bottom: 1vh;
+  letter-spacing: 0.2vh;
+}
+
+.card p {
+  text-align: left;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 2.6vh;
+  line-height: 3.6vh;
+  width: 45vh;
+}
+
+.wrapper button {
+  color: #fff;
+  letter-spacing: 0.03em;
+  font-weight: bold;
+  font-size: 2.6vh;
+  width: 100%;
+  height: 7.3vh;
+}
```

Comment:
- [ ] I acknowledge that you attempted to fix the changes required by the previous reviewer on having your project look as identical as possible to the Figma design. Just a little bit more CSS on the "see project buttons" and your project will be ready. Kindly remove the spacing between the buttons and cards.

<html>
<body>
<!--StartFragment--><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">This is how your project looks vs the figma template</p>

Figma design | Your design
-- | --
 ![image](https://user-images.githubusercontent.com/29071984/170835869-6d7c80d3-f0c4-4ed0-be5b-1a3f19700781.png) |  ![image](https://user-images.githubusercontent.com/29071984/170835847-402f282f-56de-4ef5-ac74-42f13e96e88d.png)


<!--EndFragment-->
</body>
</html>